CHANGELOG.md

```diff
@@ -1,3 +1,124 @@
+### citus v11.1.0 (September 15, 2022) ###
+
+* Adds support for PostgreSQL 15beta3
+
+* Adds a rebalancer that uses background tasks for its execution
+
+* Adds `create_distributed_table_concurrently()` which distributes tables
+  without blocking writes
+
+* Adds `citus_split_shard_by_split_points()` function
+
+* Adds support for non-blocking tenant isolation
+
+* Separates columnar storage AM into a separate logical extension
+
+* Supports logical replication in `replicate_reference_tables()`
+
+* Improves performance of blocking shard moves
+
+* Uses a faster custom copy logic for non-blocking shard moves
+
+* `isolate_tenant_to_new_shard()` adds support for columnar tables
+
+* `isolate_tenant_to_new_shard()` adds support for partitioned tables
+
+* `isolate_tenant_to_new_shard()` drops support for replicated tables
+
+* `isolate_tenant_to_new_shard()` now fails when run concurrently with itself
+
```

Comment:
We have about 6 entries for tenant isolation. Can we group them together?

Reply:
Also, these 4 entries do not start with a verb.

```suggestion
* Adds support for isolating tenants that use columnar tables

* Adds support for isolating tenants that use partitioned tables

* Drops support for isolating tenants that use replicated tables

* Prevents concurrent tenant isolation operations

```

Reply:
I realized that we already have an entry like the following on Enterprise 11.0.0 beta

> Prevents concurrent tenant isolations

https://github.com/citusdata/citus-enterprise/blob/enterprise-master/CHANGELOG.md#citus-enterprise-v1100_beta-march-22-2022

I believe this is not actually something new on Citus 11.1. Am I missing something here?

Reply:
> I realized that we already have an entry like the following on Enterprise 11.0.0 beta

You're right, I should drop this one, but not sure about the others. But it's true that others at least should be grouped.

Reply:
but will apply your suggestions for other three